packages/@react-aria/select/src/useSelect.ts

```diff
@@ -161,6 +165,9 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
         }
 
         if (props.onBlur) {
+          if (props.onFocusChange) {
+            props.onFocusChange(false);
+          }
           props.onBlur(e);
         }
```

Comment:
Mind moving this if statement outside the wrapping `if (props.onBlur)`? `onFocusChange` should still be called even if `onBlur` isn't defined

```suggestion
        if (props.onBlur) {
          props.onBlur(e);
        }
        
        if (props.onFocusChange) {
          props.onFocusChange(false);
        }
```